mava/wrappers/jaxmarl.py

```diff
@@ -280,6 +281,11 @@ def observation_spec(self) -> specs.Spec:
     def action_spec(self) -> specs.Spec:
         return jaxmarl_space_to_jumanji_spec(merge_space(self._env.action_spaces))
 
+    @cached_property
+    def min_max_action(self) -> Tuple[chex.Array, chex.Array]:
+        """Returns the minimum and maximum values from the action space"""
+        return self.action_spec().minimum, self.action_spec().maximum
+
```

Comment:
I think it would be better to leave this out and just call this in the network init when we need it, otherwise it adds another requirement to our wrappers and I think it's better to keep our wrappers as lightweight as possible :smile: 

```suggestion
```